### PR TITLE
Comments refactor

### PR DESCRIPTION
**Pull request by SemenchenkoVitaliy**
Refactor comments to achieve comments style consistency and fix grammar mistakes.
Also, I want to notice that `deprecate` and `alias` can not be called without function, although it is said that `fn` parameter is optional. Should I remove this comments or open new PR and fix this?

### CODE REVIEW

File: common.js [hunk 1]
```diff
@@ -6,19 +6,19 @@ const submodules = [
...
   'network', // Network utilities
-  'id', // Kyes and identifiers
+  'id', // Keys and identifiers
   'sort', // Sort compare functions
   'cache', // Cache (enhanced Map)
   'mp', // Metaprogramming
   'enum', // Enumerated type
   'iterator', // Iterator
-].map(path => './lib/' + path).map(require);
+].map(path => require('./lib/' + path));
```
[the remark this answers is not in the document]
Comment (SemenchenkoVitaliy) [reply common.js]: Sorry, I didn't switched to master before creating a new branch. Is it better to just remove this commit or rebase onto master and start new PR?

File: lib/callbacks.js [hunk 2]
```diff
@@ -83,11 +83,10 @@ const requiredCallback = (
 };
 
 const onceCallback = (
-  // Extracts callback and make it safe
+  // Extract callback and make it safe
   // Wrap collback with once()
```
Comment (belochub) [inline lib/callbacks.js]: s/collback/callback

File: lib/callbacks.js [hunk 3]
```diff
@@ -62,7 +62,7 @@ const unsafeCallback = (
 };
 
 const safeCallback = (
-  // Extracts callback and return common.emptiness if no callback
+  // Extract callback and return common.emptiness if there is no callback
```
Comment (belochub) [inline lib/callbacks.js]: Should we maybe move part after 'return` to the `'Returns:'` description below?

File: lib/callbacks.js [hunk 4]
```diff
@@ -73,7 +73,7 @@ const safeCallback = (
 };
 
 const requiredCallback = (
-  // Extracts callback and throw if no callback
+  // Extract callback and throw TypeError if there is no callback
```
Comment (belochub) [inline lib/callbacks.js]: Same question as above.

File: lib/data.js [hunk 5]
```diff
@@ -60,7 +60,8 @@ function cloneObject(obj) {
 }
 
 const duplicate = (
-  // Duplicate object or array (properly handles prototype and circular links)
+  // Duplicate object or array
+  // It properly handles prototype and circular links
```
Comment (belochub) [inline lib/data.js]: I think using `it` here is ambiguous, I'd prefer not to change that comment.

File: lib/math.js [hunk 6]
```diff
@@ -26,13 +26,13 @@ class CryptoRandomPrefetcher {
 const cryptoPrefetcher = (
   // Create prefetcher to use when crypto.randomBytes is required to generate
   // multiple same-size values. `bufSize` must be a multiple of `valueSize` for
-  // this to work.
+  // this to work
```
Comment (belochub) [inline lib/math.js]: Why did you remove a period here and leave it on line 53 below?
Also, I think it makes sense to have a period at the end of all full sentences when a comment has more than one sentence in it.

File: lib/mp.js [hunk 7]
```diff
@@ -1,9 +1,9 @@
 'use strict';
 
 const methods = (
-  // List method names
+  // Get list of method names
```
Comment (belochub) [inline lib/mp.js]: 'List' was a verb here, there's no need to change it.

File: lib/mp.js [hunk 8]
```diff
@@ -16,7 +16,7 @@ const methods = (
 };
 
 const properties = (
-  // List property names
+  // Get list of property names
```
Comment (belochub) [inline lib/mp.js]: Same as above.

File: lib/strings.js [hunk 9]
```diff
@@ -230,34 +231,37 @@ const arrayRegExp = (
 };
 
 const section = (
-  // Splits string by the first occurrence of separator
+  // Split string by the first occurrence of separator
   s, // string
   separator // string, or char
+  // Returns: array
```
Comment (belochub) [inline lib/strings.js]: Shouldn't it be 'array of strings' here?

File: lib/strings.js [hunk 10]
```diff
@@ -230,34 +231,37 @@ const arrayRegExp = (
...
   if (i < 0) return [s, ''];
   return [s.slice(0, i), s.slice(i + separator.length)];
 };
 
 const rsection = (
-  // Splits string by the last occurrence of separator
+  // Split string by the last occurrence of separator
   s, // string
   separator // string, or char
+  // Returns: array
```
Comment (belochub) [inline lib/strings.js]: Same question as above.

File: lib/strings.js [hunk 11]
```diff
@@ -230,34 +231,37 @@ const arrayRegExp = (
...
 };
 
 const split = (
-  // Splits string by multiple occurrence of separator
+  // Split string by multiple occurrence of separator
   s, // string
   separator = ',', // string (optional), default: ','
-  limit = -1 // number (optional), // max length of result array
+  limit = -1 // number (optional), max length of result array
+  // Returns: array
```
Comment (belochub) [inline lib/strings.js]: Same as above.

File: lib/strings.js [hunk 12]
```diff
@@ -267,10 +271,11 @@ const split = (
 );
 
 const rsplit = (
-  // Splits string by multiple occurrence of separator
+  // Split string by multiple occurrences of separator
   s, // string
   separator = ',', // string (optional), default: ','
-  limit = -1 // number (optional), // max length of result array
+  limit = -1 // number (optional), max length of result array
+  // Returns: array
```
Comment (belochub) [inline lib/strings.js]: Ditto.

File: lib/units.js [hunk 13]
```diff
@@ -81,7 +81,7 @@ const UNIT_SIZES = {
 };
 
 const sizeToBytes = (
-  // Convert string with units to int
+  // Convert string with file size to int
```
Comment (belochub) [inline lib/units.js]: This is not necessarily a file size.

Comment (belochub) [reply lib/units.js]: Also, s/int/integer, to be consistent with other places where you've changed it.

File: lib/units.js [hunk 14]
```diff
@@ -57,7 +57,7 @@ const SIZE_UNITS = [
 ];
 
 const bytesToSize = (
-  // Convert int to string size Kb, Mb, Gb and Tb
+  // Convert int to string, representing file size in Kb, Mb, Gb and Tb
```
Comment (belochub) [inline lib/units.js]: Same remarks as for the `sizeToBytes` function apply here.

Comment (belochub) [reply lib/units.js]: Also, there should probably be a comma after 'Gb'.

**Comment by tshemsedinov** [comment]
Landed in e75d985
